packages/databricks-vscode/package.json

```diff
@@ -39,6 +39,8 @@
     "activationEvents": [
         "onCommand:databricks.*",
         "onTaskType:databricks",
+        "onView:configurationView",
```

Comment:
VS Code told me that those are inferred automatically. But it this might only be for new versions so we'd have to bump the minimal VSCode version.

Reply:
Yeah. This is what the autocomplete tells me as well. But this broke our tests in an annoying way.